docs/ADMINISTRATION.md

```diff
@@ -0,0 +1,51 @@
+# Administering the Survey Portlet
+
+Use the following steps to create an survey that you can add to your portal:
```

Comment:
there is a typo ?  'an survey'

Reply:
Yep, that's I typo, I'll get that corrected.
Thanks @jgribonvald! :bow: